tests/features/event.feature

```diff
@@ -99,6 +99,8 @@ Feature: Event content creation
     Then I should see the text "Organiser name"
     And the "Internal organiser field" is not visible
 
+    And I should see the text "Media"
```

Comment:
This step looks like doesn't have big sense because on the page we have many strings like that.